### PR TITLE
Bugfix 1162

### PR DESCRIPTION
**Pull request by wdednam**
This change is [Reviewable](https://reviewable.io/reviews/highperformancecoder/minsky/162)

### CODE REVIEW

File: model/godleyTableWindow.cc [hunk 1]
```diff
@@ -201,7 +201,14 @@ namespace minsky
           {
             CairoSave cs(surface->cairo());
             cairo_move_to(surface->cairo(), x, columnButtonsOffset);
-            colWidgets[col].draw(surface->cairo());
+            // Prevent the existence of more than one equity column in single equity column mode
+            if (godleyIcon->table._assetClass(col)==GodleyAssetClass::equity && col!=godleyIcon->table.cols()-1 && !godleyIcon->table.multipleEquities) {
+               godleyIcon->table.deleteCol(col);
+               colWidgets[col].draw(surface->cairo());
+		   }
+            else {
```
Comment (highperformancecoder) [inline model/godleyTableWindow.cc]: I'm not quite sure what you're doing here, but seems unrelated to #1162. Could you please remove this and commit in a separate PR if so.

Comment (wdednam) [reply model/godleyTableWindow.cc]: Yes, this somehow snuck in to this PR, even after I had pulled from remote/master?. Sorry, I'll remove it right away, it is part of the multiple equity column PR.
